src/main/java/frc/trigon/robot/subsystems/coralintake/CoralIntake.java

```diff
@@ -63,29 +64,31 @@ public void updateMechanism() {
     public void updatePeriodically() {
         intakeMotor.update();
         funnelMotor.update();
-        angleMotor.update();
-        encoder.update();
+        elevatorMotor.update();
         beamBreak.updateSensor();
     }
 
     @Override
     public SysIdRoutine.Config getSysIdConfig() {
-        return CoralIntakeConstants.ANGLE_SYSID_CONFIG;
+        return CoralIntakeConstants.ELEVATOR_SYSID_CONFIG;
     }
 
     @Override
     public void stop() {
         intakeMotor.stopMotor();
         funnelMotor.stopMotor();
-        angleMotor.stopMotor();
+        elevatorMotor.stopMotor();
     }
 
-    public boolean atTargetState() {
-        return Math.abs(getCurrentEncoderAngle().minus(targetState.targetAngle).getDegrees()) < CoralIntakeConstants.ANGLE_TOLERANCE.getDegrees();
+    public boolean atState(CoralIntakeConstants.CoralIntakeState targetState) {
+        return targetState == this.targetState && atTargetAngle();
     }
 
-    public boolean atState(CoralIntakeConstants.CoralIntakeState targetState) {
-        return targetState == this.targetState && atTargetState();
+    public boolean atTargetAngle() {
+        final double
+                elevatorDifferenceFromTargetStateRotations = Math.abs(getCurrentElevatorPositionRotations() - targetState.targetPositionRotations),
+                positionToleranceRotations = metersToRotations(CoralIntakeConstants.POSITION_TOLERANCE_METERS);
```

Comment:
just do `final double` twice
save the tolerance in rotations (with a conversion to rotations)